api/tests/mocha/ddoc-extraction.spec.js

```diff
@@ -37,35 +47,86 @@ describe('DDoc extraction', () => {
     };
 
     const getDdoc = get.withArgs('_design/medic').resolves(ddoc);
-    const getDdocAttachment = getAttachment
-      .withArgs('_design/medic', 'ddocs/compiled.json')
-      .resolves(Buffer.from(JSON.stringify(attachment)));
-    const getNew = get.withArgs('_design/new').rejects({ status: 404 });
-    const getUpdated = get.withArgs('_design/updated').resolves({
-      _id: '_design/updated', _rev: '1', views: { doc_by_valed: { map: 'function() { return true; }' } }
+    const getMedicAttachment = getAttachment
+      .withArgs('_design/medic', 'ddocs/medic.json')
+      .resolves(Buffer.from(JSON.stringify(medicAttachment)));
+    const getSentielAttachment = getAttachment
+      .withArgs('_design/medic', 'ddocs/sentinel.json')
+      .resolves(Buffer.from(JSON.stringify(sentinelAttachment)));
+    const getUsersMetaAttachment = getAttachment
+      .withArgs('_design/medic', 'ddocs/users-meta.json')
+      .resolves(Buffer.from(JSON.stringify(usersMetaAttachment)));
+    
+    const getMedicNew = get.withArgs('_design/medic-new').rejects({ status: 404 });
+    const getMedicUpdated = get.withArgs('_design/medic-updated').resolves({
+      _id: '_design/medic-updated', _rev: '1', views: { doc_by_valed: { map: 'function() { return true; }' } }
     });
-    const getUnchanged = get.withArgs('_design/unchanged').resolves({
-      _id: '_design/unchanged', _rev: '1', views: { doc_by_valid: { map: 'function() { return true; }' } }
+    const getMedicUnchanged = get.withArgs('_design/medic-unchanged').resolves({
+      _id: '_design/medic-unchanged', _rev: '1', views: { doc_by_valid: { map: 'function() { return true; }' } }
     });
+
+    const getSentinelNew = get.withArgs('_design/sentinel-new').rejects({ status: 404 });
```

Comment:
`get` is `const get = sinon.stub(db.medic, 'get');` . The `sentinel` ddoc should exist in `medic-sentinel`, right?

Reply:
Yes absolutely. Same problem in the `src` code. Fixed in both places.